services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceTest.kt

```diff
@@ -93,6 +94,7 @@ class HakemusServiceTest {
     private val paatosService: PaatosService = mockk()
     private val publisher: ApplicationEventPublisher = mockk()
     private val tormaystarkasteluLaskentaService: TormaystarkasteluLaskentaService = mockk()
+    private val permissionService: PermissionService = mockk()
```

Comment:
This seems to be unnecessary?

Reply:
Thank you for noticing. This was left over from an earlier iteration of getting the permissions for a hankekayttaja. Removed.